flink-connectors/flink-connector-rabbitmq/src/test/java/org/apache/flink/streaming/connectors/rabbitmq/RMQSourceTest.java

```diff
@@ -86,13 +86,23 @@
 
 	private volatile Exception exception;
 
-	@Before
-	public void beforeTest() throws Exception {
-
+	/**
+	 * Get a mock context for initializing the source's state.
+	 *
+	 * @return a mocked context that can be used in {@link org.apache.flink.streaming.api.checkpoint.CheckpointedFunction#initializeState}
```

Comment:
Maybe that's just my style but I often prefer Javadoc without a `@return` clause if it's a duplicate of the main text. Here the main text can just be `Gets a mock context for initializing the source's state via {@link org.apache.flink.streaming.api.checkpoint.CheckpointedFunction#initializeState}.`

Reply:
Makes sense to me, will update.